specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/SessionPools.json

```diff
@@ -0,0 +1,702 @@
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SessionPoolNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Session pool delete has been started.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "Session pool does not exist."
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Delete Session Pool": {
+            "$ref": "./examples/SessionPools_Delete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ScaleConfiguration": {
+      "description": "Scale configuration.",
+      "type": "object",
+      "properties": {
+        "maxConcurrentSessions": {
+          "description": "The maximum count of sessions at the same time.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "readySessionInstances": {
+          "description": "The minimum count of ready session instances.",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "Secret": {
+      "description": "Secret definition.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Secret Name.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Secret Value.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ],
+          "x-ms-secret": true
+        }
+      }
+    },
+    "DynamicPoolConfiguration": {
+      "description": "Dynamic pool configuration.",
+      "type": "object",
+      "properties": {
+        "executionType": {
+          "description": "The execution type of the session pool.",
+          "enum": [
+            "Timed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ExecutionType",
+            "modelAsString": true
+          }
+        },
+        "cooldownPeriodInSeconds": {
+          "description": "The cooldown period of a session in seconds.",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "SessionRegistryCredentials": {
+      "description": "Session pool Private Registry Credentials.",
```

Comment:
>Private Registry Credentials [](http://example.com/codeflow?start=35&length=28)

nit, can avoid some excessive capitalization in docs

Reply:
Fixed, went through the file and fixed some other similar docs, too